app/views/articles/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= render 'templates/navbar' %>
```

Comment:
Rendering your navbar is repeated in several templates.
What do you think of pulling this into your `views/layouts/application.html.erb`?

Reply:
Ahhh, that's a much smarter way of doing it